### ISSUE TITLE
Generated code for HTTP contains wrong host when url is given by variable

### ISSUE THREAD

**Issue by mjhcorporate** [a85/POSTMan-Chrome-Extension#8725]
**Describe the bug**
Let's say, I'm creating a new GET-request in Postman, and I set the request-url to `https://github.com/postmanlabs/postman-app-support/issues`. Then I can click on "Code" and view the corresponding HTTP code:

```
GET /postmanlabs/postman-app-support/issues HTTP/1.1
Host: github.com
```

However, when I introduce a variable "fullUrlVarariable = https://github.com/postmanlabs/postman-app-support/issues` and set the request-url to `{{fullUrlVariable}}`, then the "Generated code for HTTP" has a suprising path/host combination: 

```
GET / HTTP/1.1
Host: https://github.com/postmanlabs/postman-app-support/issues
```
I used wireshark to verify that this is _not_ what is being sent over the wire - the _actual_ http-packet had the host-field set to `Host: github.com`.

**To Reproduce**
1. Create an environment name `fullUrlVariable`, and set it to a url that includes a path and protocol, e.g. `https://github.com/postmanlabs/postman-app-support/issues`
2. Create a new Request
3. Set the request-url to `{{fullUrlVariable}}´
4. Click on "Code" (just under the "Save" button)
5. Choose HTTP
6. Observe that the host-field not only contains the host, but (suprisingly) the protocol and the path as well.

**Expected behavior**
The protocol and path should not be part of the "host" field, the path should be given after the http-verb instead. 

**Screenshots**
Unexpected host-field:
![image](https://user-images.githubusercontent.com/49681105/86117501-71e43800-bacf-11ea-8b59-2b88c4ea09b7.png)

**App information (please complete the following information):**
 - App Type: Native
 - Postman Version 7.27.1
 - OS: Windows 100.018363

**Comment by webholik** [comment]
Fixed by https://github.com/postmanlabs/postman-code-generators/pull/397

**Comment by umeshp7** [comment]
@mjhcorporate Hey! Looks like this has been fixed in the recent version of app 7.36. Closing this for now feel free to reopen this if the issue persists.

**Comment by mj-h** [comment]
@umeshp7 : I can confirm that the issue is fixed in 7.36. Thank you!

**Comment by umeshp7** [comment]
@mj-h Great! Thank you for the confirmation!
